lib/datadog/ci/contrib/cucumber/step.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Datadog
+  module CI
+    module Contrib
+      module Cucumber
+        # instruments Cucumber::Core::Test::Step from cucumber-ruby-core to change
+        module Step
+          def self.included(base)
+            base.prepend(InstanceMethods)
+          end
+
+          module InstanceMethods
+            def execute(*args)
+              test_span = CI.active_test
+              if test_span&.skipped_by_itr?
+                @action.skip(*args)
```

Comment:
I'm guessing Cucumber doesn't have a way to report the message `CI::Ext::Test::ITR_TEST_SKIP_REASON`, unlike Minitest and RSpec.

Reply:
Yep, at least not easily findable.

`skip_this_scenario` method does not accept arguments with skip reason for example